digitalpy/core/zmanager/impl/default_routing_worker.py

```diff
@@ -94,9 +94,11 @@
             response (Response): the response to be sent to integration_manager
             protocol (str): the protocol of the message
         """
-        self.formatter.serialize(response)
-        message = protocol.encode()+b","+response.get_values()
-        self.integration_manager_sock.send(message)
+        response_topics = self.get_response_topic(response, protocol)
+        #self.formatter.serialize(response)
+        #response_value = response.get_values()
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>E265 block comment should start with '# '